contracts/payloads/IGP85/PayloadIGP85.sol

```diff
@@ -214,6 +205,80 @@ contract PayloadIGP85 is PayloadIGPMain {
         }
     }
 
+    // @notice Action 5: Set Launch Limits for USD0-USDC & fxUSD-USDC
+    function action5() internal isActionSkippable(5){
+        
+        {
+            address USD0_USDC_DEX = getDexAddress(23);
+            // USD0-USDC DEX
+            {
+                // USD0-USDC Dex
+                Dex memory DEX_USD0_USDC = Dex({
+                    dex: USD0_USDC_DEX,
+                    tokenA: USD0_ADDRESS,
+                    tokenB: USDC_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_USD0_USDC); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(USD0_USDC_DEX, TEAM_MULTISIG, false);
+            }
+
+            IFluidDex(USD0_USDC_DEX).updateMaxSupplyShares(
+                    7_500_000 * 1e18 // $15M
+            );
+        }
+
+        {
+            address fxUSD_USDC_DEX = getDexAddress(24);
+            // fxUSD-USDC DEX
+            {
+                // fxUSD-USDC Dex
+                Dex memory DEX_fxUSD_USDC = Dex({
+                    dex: fxUSD_USDC_DEX,
+                    tokenA: fxUSD_ADDRESS,
+                    tokenB: USDC_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10_000_000, // $10M
+                    baseBorrowLimitInUSD: 0, // $0
+                    maxBorrowLimitInUSD: 0 // $0
+                });
+                setDexLimits(DEX_fxUSD_USDC); // Smart Collateral
+
+                DEX_FACTORY.setDexAuth(fxUSD_USDC_DEX, TEAM_MULTISIG, false);
+            }
+
+            IFluidDex(fxUSD_USDC_DEX).updateMaxSupplyShares(
+                    7_500_000 * 1e18 // $15M
+            );
+        }
+    }
+    // @notice Action 6: Set Rebalancers for USD0-USDC & fxUSD-USDC
+    function action6() internal isActionSkippable(6){
+        {
+            address fSL21_USD0_USDC = getDexAddress(23);
+
+            // set rebalancer at fSL21 to reserve contract proxy
+            IFTokenAdmin(fSL21_USD0_USDC).updateRebalancer(
+                0xD2245ee5C3099d65a3d0fdCecA0f71Cc4aA8f0FF
```

Comment:
for both here, rebalancer is set to reserve contract: 0x264786EF916af64a1DB19F513F24a3681734ce92

you are triggering on the SmartLending contract

need to create a new method `getSmartLendingAddress` which calls this method on the SmartLendingFactory https://github.com/Instadapp/fluid-contracts-public/blob/main/contracts/protocols/dex/smartLending/factory/main.sol#L205. SmartLendingFactory is `0xe57227C7d5900165344b190fc7aa580bceb53B9B`, passing in the dexId

Reply:
ohh i actually just did get address and set the addresses that were in the slack messages as the rebalancers
let me update accordingly